TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>So the fluxes aren't exactly the same. A clue that might help us solve this mystery: all of the errors are a multiple of 1/64. Even better, they're exclusively +- 1, 2, 4 / 64.</p><p> </p><p>To me, this screams, "something wrong is happening in a binary calculation somewhere". </p><p> </p><p>Update: GAAHHHHHHHHH. That's exactly what's going on here. If you redo this calculation with 64-bit floats, the errors "spread out" into their true values. I think what's happening here is that the TESS pipeline sums up the flux with a 64-bit calculation, but only gives us a 32-bit result.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>sorry, I'm not sure how to re-do this. I tried changing the data types to floats (Python floats are already 64 bit), but that made no difference to the plot. </p>